### DISCUSSION TITLE
Using "Nginx Proxy Manager" instead of "Nginx HA SSL proxy"

### DISCUSSION THREAD

**Discussion by little-penguin88** [llamafilm/tesla-http-proxy-addon#36] [Q&A]
Is it possible to use the Nginx Proxy Manager community plugin instead of the Nginx SSL proxy for this tool? If so can anyone help me with the right Host settings I need to put into to expose the .pem file and get the right /callback settings to get it running? Any help would be much appreciated

Thank you

**Comment by baylanger** [comment]
Cloudflared add-on supports Nginx Proxy Manager

https://github.com/brenner-tobias/addon-cloudflared/blob/main/cloudflared/DOCS.md#option-nginx_proxy_manager

**Reply by little-penguin88** [reply]
Ok, but this does not explain how to use the Tesla proxy with the nginx proxy manager instead of the nginx SSL Addon, does it?

**Reply by baylanger** [reply]
Sorry I’m away on my phone…  there’s a [Nginx config ](https://github.com/llamafilm/tesla-http-proxy-addon/blob/main/tesla_http_proxy/app/nginx_tesla.conf) file available that you can try to use to replicate the config within NPM.

**Reply by florianfish** [reply]
hello i'm using npm from HACS. @little-penguin88 have you success with it to expose ssl certificate url ?

**Comment by little-penguin88** [comment]
Actually I switched to the Home assistant Nginx SSL proxy

Florian Fish ***@***.***> schrieb am Mi., 10. Apr. 2024,
11:09:

**Comment by florianfish** [comment]
@little-penguin88 you use it with duckduck ?

**Comment by little-penguin88** [comment]
Yes

Florian Fish ***@***.***> schrieb am Mi., 10. Apr. 2024,
11:22:

**Comment by florianfish** [comment]
Can you share your nginx conf ? I will try with my HA. Thanks!

**Comment by little-penguin88** [comment]
I just followed the instructions to set it up step by step. The Nginx
config is being updated by the addon during the initial run.

Florian Fish ***@***.***> schrieb am Mi., 10. Apr. 2024,
13:10:
